### ISSUE TITLE
AttributeError: module 'collections' has no attribute 'MutableSet'

### ISSUE THREAD

**Issue by EeroKH** [bit-team/backintime#1210]
I am using arch linux. 
When starting back in time from icon nothing happens.
when starting on command line I get following erros;

$ backintime
Traceback (most recent call last):
  File "/usr/share/backintime/common/backintime.py", line 27, in <module>
    import config
  File "/usr/share/backintime/common/config.py", line 32, in <module>
    import tools
  File "/usr/share/backintime/common/tools.py", line 1801, in <module>
    class OrderedSet(collections.MutableSet):
AttributeError: module 'collections' has no attribute 'MutableSet'

I just got Python version 3.10. 

Thanks 
- Eero -

**Comment by RJDan** [comment]
I am having the same issue after a recent upgrade on Ubuntu 22.04
I found this: 
https://github.com/vm03/payload_dumper/issues/27

If you are using python 3.10, that might be the cause.

```
$ python --version
Python 2.7.18
$ python3 --version
Python 3.10.2
$ hostnamectl 
Operating System: Ubuntu Jammy Jellyfish (development branch)
          Kernel: Linux 5.15.0-22-generic
    Architecture: x86-64
 Hardware Vendor: ASUSTeK COMPUTER INC.
  Hardware Model: ASUS TUF Gaming A15 FA506IH_FA506IH
```

**Comment by emtiu** [comment]
I think #1174 is meant to fix this, but I'm unsure whether this project has any maitainers at the moment.

**Comment by Germar** [comment]
This is fixed in 1.3.1 but Ubuntu/Debian did not pick up that release yet. Please install from PPA:
```
sudo add-apt-repository ppa:bit-team/stable
```

**Comment by RJDan** [comment]
Hi

That hasn't helped my situation. 
I was on the testing PPA already.
I have now removed the testing PPA and added the stable. Removed BIT and reinstalled it. Nothing has changed with the problem after this.
I have 1.3.1 installed now.
